build_tools/packaging/python/templates/rocm/src/rocm_sdk/_dist_info.py

```diff
@@ -136,6 +136,7 @@ def determine_target_family() -> str:
         target_family = discover_current_target_family()
         if target_family is None:
             target_family = DEFAULT_TARGET_FAMILY
+            print(f"[INFO] Using default target family: {DEFAULT_TARGET_FAMILY}")
```

Comment:
We should be careful about logging here. This runs for simple things like `import torch`. Let's make sure to fix at the root instead of just adding more logging.

I added some ideas to https://github.com/ROCm/TheRock/issues/1421. I think with the changes I tested there we can also include these logs.